cfsctl/command/rename_command.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var (
+	renameOld string
+	renameNew string
+)
+
+var renameCmd = &cobra.Command{
+	Use:   "rename",
+	Short: "rename a file on a cfs node",
+	Long:  "",
+	Run: func(cmd *cobra.Command, args []string) {
+		rawHandle(context.TODO(), handleRename)
+	},
+}
+
+func init() {
+	renameCmd.PersistentFlags().StringVarP(&renameOld, "oldname", "", "", "old name")
```

Comment:
@yunxing  any example of the format you prefer?
@xiang90  I referenced sub pkg structure from `etcd`, do you want them in the same pkg for now?